packages/hash/frontend/src/pages/[account-slug]/types/entity-type/entity-type-tabs.tsx

```diff
@@ -1,28 +1,82 @@
-import { EntityType } from "@blockprotocol/type-system-web";
 import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@hashintel/hash-design-system";
-import { Box, Typography } from "@mui/material";
-import { EntityTypeDefinitionTab } from "./entity-type-definition-tab";
+import { Box, Tabs, tabsClasses } from "@mui/material";
+import { useRouter } from "next/router";
+import { useMemo } from "react";
+import { useFormContext, useWatch } from "react-hook-form";
+import { EntityTypeEditorForm } from "./form-types";
 import { TabButton } from "./tab-button";
+import { TabLink } from "./tab-link";
+import { useEntityTypeEntities } from "./use-entity-type-entities";
+
+export const EntityTypeTabs = () => {
+  const router = useRouter();
+
+  const { control } = useFormContext<EntityTypeEditorForm>();
+  const propertiesCount = useWatch({ control, name: "properties.length" });
+
+  const { entities } = useEntityTypeEntities() ?? {};
+
+  const baseUri = useMemo(
+    () =>
+      `/${router.query["account-slug"]}/types/entity-type/${router.query["entity-type-id"]}`,
+    [router.query],
+  );
+
+  const currentTab = router.query.tab ?? "";
 
-export const EntityTypeTabs = ({ entityType }: { entityType: EntityType }) => {
   return (
-    <Box display="flex">
-      <EntityTypeDefinitionTab entityType={entityType} />
+    <Box display="flex" overflow="visible">
+      <Tabs
```

Comment:
I'd be happy with the ripple if it gradient-faded out to white, rather than exposing ugly sharp box corners on the element. If we can fix that, we should - and this would be my preference.

Otherwise we can lose it.

Reply:
How does this look?

![ripple](https://user-images.githubusercontent.com/37453800/202580824-50e4585b-a50b-4083-ab8c-fcb3ca3defd0.gif)

Reply:
I think this works with no ripple.